### DISCUSSION TITLE
Call to a member function determineOrderColumnName() on null

### DISCUSSION THREAD

**Discussion by faguvenli** [spatie/laravel-medialibrary#3249] [General]
while trying to upload images using filamentphp, I'm having this error after pressing the create / update button.
this is how I use it:
SpatieMediaLibraryFileUpload::make('attachments')
                        ->multiple()
                        ->enableReordering()
                        ->responsiveImages()
                        ->image()
                        ->visibility('public')
                        ->maxSize(1024)
                        ->imagePreviewHeight(150)
                        ->disk('image_path')

**Comment by almontasser** [comment]
It looks like it got fixed, try updating your packages
```sh
composer update
```
